controllers/url.go

```diff
@@ -13,46 +13,69 @@ import (
 )
 
 func CreateTinyURL(ctx *gin.Context, db *bun.DB) {
-	var body models.Tinyurl
+    var body models.Tinyurl
 
-	if err := ctx.BindJSON(&body); err != nil {
-		ctx.JSON(http.StatusBadRequest, dtos.URLCreationResponse{
-			Message: "Invalid JSON format: " + err.Error(),
-		})
-		return
-	}
+    if err := ctx.BindJSON(&body); err != nil {
+        ctx.JSON(http.StatusBadRequest, dtos.URLCreationResponse{
+            Message: "Invalid JSON format: " + err.Error(),
+        })
+        return
+    }
 
-	if body.OriginalUrl == "" {
-		ctx.JSON(http.StatusBadRequest, dtos.URLCreationResponse{
-			Message: "original url is required",
-		})
-		return
-	}
+    if body.OriginalUrl == "" {
+        ctx.JSON(http.StatusBadRequest, dtos.URLCreationResponse{
+            Message: "Original URL is required",
+        })
+        return
+    }
 
-	if err := db.NewSelect().Model(&body).Where("original_url = ?", body.OriginalUrl).Scan(ctx, &body); err == nil {
-		ctx.JSON(http.StatusOK, dtos.URLCreationResponse{
-			Message:  "Tiny URL already exists",
-			ShortURL: body.ShortUrl,
-		})
-		return
-	}
+    if body.ShortUrl != "" {
+        if len(body.ShortUrl) < 5 {
+            ctx.JSON(http.StatusBadRequest, dtos.URLCreationResponse{
+                Message: "Custom short URL must be at least 5 characters long",
+            })
+            return
+        }
```

Comment:
why there is a limit to the length of the shorturl?

Reply:
Currently, every user can generate short URLs and custom URLs so make it uniform random generate and custom short URL have length 5